src/uct/api/uct_def.h

```diff
@@ -111,6 +117,13 @@ typedef int                      uct_worker_cb_id_t;
     |<---- stride ----->|<---- stride ----->|
    @endverbatim
  *
+ * In order to describe an interleaved memory layout, each structure contains
```

Comment:
can we encode the latter example as:
dtype C {
  A_ (count:1)
  B_ (count:2)
}
send(buffer, dtype=C, count=2)